docs/src/main/paradox/cluster-usage.md

```diff
@@ -143,7 +143,7 @@ Java
 
 
 If you find it inconvenient to handle the `CurrentClusterState` you can use
-@scala[@scaladoc[ClusterEvent.InitialStateAsEvents](cluster.ClusterEvent$$InitialStateAsEvents$)] @java[@javadoc[ClusterEvent.initialStateAsEvents()](pekko.cluster.ClusterEvent#initialStateAsEvents())] as parameter to @apidoc[subscribe](pekko.cluster.Cluster) {scala="#subscribe(subscriber:org.apache.pekko.actor.ActorRef,to:Class[_]*):Unit" java="#subscribe(org.apache.pekko.actor.ActorRef,org.apache.pekko.cluster.ClusterEvent.SubscriptionInitialStateMode,java.lang.Class...)"}.
+@scala[@scaladoc[ClusterEvent.InitialStateAsEvents](org.apache.pekko.cluster.ClusterEvent$$InitialStateAsEvents$)] @java[@javadoc[ClusterEvent.initialStateAsEvents()](pekko.cluster.ClusterEvent#initialStateAsEvents())] as parameter to @apidoc[subscribe](pekko.cluster.Cluster) {scala="#subscribe(subscriber:org.apache.pekko.actor.ActorRef,to:Class[_]*):Unit" java="#subscribe(org.apache.pekko.actor.ActorRef,org.apache.pekko.cluster.ClusterEvent.SubscriptionInitialStateMode,java.lang.Class...)"}.
```

Comment:
not worth delaying the merge over - but `pekko...` should work - the `@java` link on this line works -- there are a few more changes to use `org.apache.pekko` elsewhere in this PR that could also be changed

Reply:
I ended up checking the source code of paradox, something weird is going on where `pekko` works in some cases and not in others. We can always check this later but I just want to fix the links for now (and at least with this PR we can actually check for link regressions in the future).